core/src/main/scala/org/apache/spark/eventhubs/EventHubsConf.scala

```diff
@@ -368,6 +374,13 @@ final class EventHubsConf private (private val connectionStr: String)
     self.get(OperationTimeoutKey) map (str => Duration.parse(str))
   }
 
+  /** it is the same as operationTimeout.
```

Comment:
Nit - wording:

/** This is the exact same thing as operationTimeout, the only difference is in the types
  * (Java Duration vs Scala FiniteDuration)
  */

Reply:
Thanks, fixed.